qiskit/circuit/library/standard_gates/multi_control_rotation_gates.py

```diff
@@ -205,6 +205,86 @@ def _mcsu2_real_diagonal(
     return circuit
 
 
+def _mcsu2_with_params(
+    unitary_type: str,  #'rz', 'ry' or 'rz'
+    theta: ParameterValueType,
+    num_controls: int,
+    ctrl_state: Optional[str] = None,
+) -> QuantumCircuit:
+    """
+    Return a parametrized multi-controlled RX or RY or RZ gate [1]_.
+
+    Args:
+        unitary_type: SU(2) unitary matrix type: 'rx' or 'ry' or 'rz'.
+        num_controls: The number of control qubits.
+        ctrl_state: The state on which the SU(2) operation is controlled. Defaults to all
+            control qubits being in state 1.
+
+    Returns:
+        A :class:`.QuantumCircuit` implementing the multi-controlled RX / RY / RZ gate.
+
+    References:
+
+        .. [1]: R. Vale et al. Decomposition of Multi-controlled Special Unitary Single-Qubit Gates
+            `arXiv:2302.06377 (2023) <https://arxiv.org/abs/2302.06377>`__
+
+    """
+    from .x import MCXVChain
+    from qiskit.circuit.library.standard_gates.ry import RYGate
+    from qiskit.circuit.library.standard_gates.rz import RZGate
+
+    if unitary_type == "ry":
+        s_gate = RYGate(-theta / 4)
+    elif unitary_type in ("rz", "rx"):
+        s_gate = RZGate(-theta / 4)
```

Comment:
Is the `s_gate` for `RXGate` also `RXGate(-theta / 4)`?

Reply:
Yes, the difference is at lines 258 & 282:
```
   if unitary_type == "rx":
        circuit.h(target)
```